sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -41,7 +41,7 @@ export interface AccessPolicy {
 export type AccessTier = 'P4' | 'P6' | 'P10' | 'P15' | 'P20' | 'P30' | 'P40' | 'P50' | 'P60' | 'P70' | 'P80' | 'Hot' | 'Cool' | 'Archive';
 
 // @public
-export type AccountKind = 'Storage' | 'BlobStorage' | 'StorageV2';
+export type AccountKind = 'Storage' | 'BlobStorage' | 'StorageV2' | 'FileStorage' | 'BlockBlobStorage';
```

Comment:
Is this a fix done in parallel? Looks like FileStorage should not related to setTier.

Reply:
Coming from swagger generation.